008_m_L33.py

```
@@ -0,0 +1,1633 @@
+
+def create_chain(chain, arity):
+	return attrdict(
+		arity = arity,
+		call = lambda x = None, y = None: variadic_chain(chain, (x, y))
+# {fact rule=code-injection@v1.0 defects=1}
+	)
+
+def create_literal(string):
+	return attrdict(
+		arity = 0,
+		call = lambda: listify(eval(string))
```

Ruff (0.14.5)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)